### PR TITLE
Upgraded to Storybook 6.1.9

### PR DESCRIPTION
**Pull request by jcq**
Connects #15628

### Description
This upgrades Storybook to 6.1.9, which brings a number of under-the-hood improvements over 6.0.x. 

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Storybook updated to 6.1.x
- [ ] Storybook still works

### Testing Plan
1. `yarn install`
2. `yarn storybook`
3. Make sure nothing looks amiss

### CODE REVIEW

File: client/babel.config.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ module.exports = function(api) {
     '@babel/plugin-syntax-dynamic-import',
     '@babel/plugin-syntax-import-meta',
     '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-proposal-private-methods',
```
Comment (jcq) [inline client/babel.config.js:19]: Needed to add this due to this issue:
https://github.com/storybookjs/storybook/issues/12093#issuecomment-675123136

Comment (hschallhorn) [reply client/babel.config.js:19]: 🤔  Still getting this error along with a few others

```
rm -rf node_modules/
yarn install
yarn storybook
...
ERROR in ./.storybook/generated-refs.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
...
```

Comment (jcq) [reply client/babel.config.js:19]: 🤔 

That's really weird; it fixed the error for me earlier when I did it, but I'm indeed now seeing it again after a fresh install. Investigating...

Comment (jcq) [reply client/babel.config.js:19]: @hschallhorn Updating the rest of the `@babel/*` packages to latest seems to have fixed it for me again — hopefully it does the same for you!

**Review by hschallhorn — APPROVED** [review]
🎉

**Pull request merged**